webapps/frontend/app/viewmodels/wizards/addpreset/gather.js

```diff
@@ -81,10 +81,10 @@ define([
             var newID = Math.max(0, Math.max.apply(this, self.data.policies().map(function(item) { return item.id(); }))) + 1;
             self.data.policies.push({
                 id: ko.observable(newID),
-                k: ko.observable(0).extend({ numeric: { min: 1 } }),
-                m: ko.observable(0).extend({ numeric: { min: 0 } }),
-                c: ko.observable(0).extend({ numeric: { min: 1 } }),
-                x: ko.observable(0).extend({ numeric: { min: 1 } })
+                k: ko.observable(),
```

Comment:
You removed the numeric extension here, but it's still added in the index.js. Why was it removed in the first place, it does all integer parsing automatically. Removing this will most likely give some unwanted results if people start entering non numeric characters.